pkg/workload/workload.go

```diff
@@ -237,3 +237,31 @@ func UpdateStatusIfChanged(ctx context.Context,
 	// Updating an existing condition
 	return UpdateStatus(ctx, c, wl, conditionType, conditionStatus, reason, message)
 }
+
+// AdmissionFrom returns only the fields necessary for admission using
+// ServerSideApply.
+func AdmissionFrom(w *kueue.Workload) *kueue.Workload {
```

Comment:
let's take the opportunity to have a better name for this:

```suggestion
func AdmissionPatch(w *kueue.Workload) *kueue.Workload {
```

Reply:
Done